yellowbrick/features/rankd.py

```diff
@@ -449,14 +449,14 @@ class Rank2D(RankDBase):
         'covariance': lambda X: np.cov(X.transpose()),
     }
 
-    def __init__(self, ax=None, algorithm='pearson', features=None,
+    def __init__(self, ax=None, algorithm='pearson', labels=None,
                  colormap='RdBu_r', show_feature_names=True, **kwargs):
         """
         Initialize the class with the options required to rank and
-        order features as well as visualize the result.
+        order labels as well as visualize the result.
```

Comment:
Here this is actually features that are being ranked, so does not need to be changed.